worlds/witness/data/static_items.py

```diff
@@ -41,7 +41,19 @@ def populate_items() -> None:
             ITEM_GROUPS.setdefault("Symbols", set()).add(item_name)
         elif definition.category is ItemCategory.DOOR:
             classification = ItemClassification.progression
-            ITEM_GROUPS.setdefault("Doors", set()).add(item_name)
+
+            first_entity_hex = cast(DoorItemDefinition, definition).panel_id_hexes[0]
+            entity_type = static_witness_logic.ENTITIES_BY_HEX[first_entity_hex]["entityType"]
+
+            if entity_type == "Door":
+                ITEM_GROUPS.setdefault("Doors", set()).add(item_name)
+            elif entity_type == "Panel":
+                ITEM_GROUPS.setdefault("Panel Keys", set()).add(item_name)
+            elif entity_type in {"EP", "Obelisk Side", "Obelisk"}:
```

Comment:
Why are EPs considered Obelisk Key?

Reply:
It's about which type of entity the item **unlocks**

If an item unlocks an EP, it is an Obelisk Key